array/array.mbt

```diff
@@ -289,3 +342,175 @@ pub fn swap[T](self : Array[T], i : Int, j : Int) -> Unit {
   self[i] = self[j]
   self[j] = temp
 }
+
+test "swap" {
+  let arr = [1, 2, 3, 4, 5]
+  arr.swap(0, 1)
+  @assertion.assert_eq(arr, [2, 1, 3, 4, 5])?
+}
+
+/// Check if all the elements in the array match the condition.
+/// 
+/// # Example 
+/// 
+/// ```
+/// let arr = [1, 2, 3, 4, 5]
+/// arr.all(fn(ele) { ele < 6 }) // true
+/// arr.all(fn(ele) { ele < 5 }) // false
+/// ```
+pub fn all[T](self : Array[T], f : (T) -> Bool) -> Bool {
+  for i = 0; i < self.length(); i = i + 1 {
+    if f(self[i]) == false {
```

Comment:
You can use `not` function. See the [doc](https://www.moonbitlang.com/docs/syntax#boolean) for reference